server/lyft/decorators/audit_project_commands_wrapper.go

```diff
@@ -0,0 +1,148 @@
+package decorators
+
+import (
+	"encoding/json"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/metrics"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/lyft/aws/sns"
+)
+
+// AuditProjectCommandWrapper is a decorator that notifies sns topic
+// about the state of the command. It is used for auditing purposes
+type AuditProjectCommandWrapper struct {
+	SnsWriter sns.Writer
+	events.ProjectCommandRunner
+}
+
+func (p *AuditProjectCommandWrapper) Apply(ctx models.ProjectCommandContext) models.ProjectResult {
+	ctx.SetScope("audit_applies")
```

Comment:
I actually think this method is broken.  Maybe just use `scope.Scope(..)`

Also, let's think about the namespacing here. The current scope should be something like:
`production.app.atlantis.cmd.comment.apply.project...` at this point.

Not sure adding audit_applies as namespacing here makes sense since we already know that its an apply.  Also, Atlantis isn't doing the auditing right? So it's kind of confusing to use that word in the metric. This is just writing the job to an sns topic.    

That being said, do we really care about tracking this kind of thing at the project level? I can't think of usecases, but let me know if you've thought of any.  In my mind I just want to know the failure rate of posting to the sns topic tbh and any latencies that result cuz of it.

so like even just grouping our aws related metrics under one top level heading would be good enough for me. ie.

`production.app.atlantis.aws.sns.<success/error/latency>`

If we go this route, then the stats can just live in the sns writer itself and don't need to be here since we already track errors and success in the instrumented project command runner.



when searching for metrics intuitively I'd like to see:

`production.app.atlantis.cmd.comment.apply.project.`